### ISSUE TITLE
inaccessible JS file

### ISSUE THREAD

**Issue by darkenvy** [wikimedia/openzim#10]
Every other JS file that is referenced in HTML with a script tag is fine. However, in javascript, if you decide to request a file locally (using XHR related methods), you are stuck requesting from "A/". 

You cannot specify a root path in this XHR request because the root path always leads to the kiwix-serve home (not the individual zim's home). Therefore we are stuck referencing the filepath relatively. eg: "/assets/iFrameScripts-dff78024c9.js".

TL;DR: because this is a asset file (not html), it is placed in "-/" where it belongs. However, javascript XHR can only relatively point to "A/" and deeper. 

PS: I don't have a solution. But XHR in javascript is common. And ripping (CC/PublicDomain) sites use this often and makes a pain out of this process.

**Comment by kelson42** [comment]
@darkenvy This repo is deprecated, please do not use it. It's true the namespaces bring problems and this is a know problem. But you can workaround it by just writting/anticipating the correct path in the javascript/html. For now nothing much more we can do.

**Comment by kelson42** [comment]
This ticket should be closed.
